CONTRIBUTING-fa_IR.md

```diff
@@ -1,4 +1,4 @@
-*این متن را در زبان‌های دیگر بخوانید: [English](CONTRIBUTING.md), [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
+*این متن را در زبان‌های دیگر بخوانید: [Deutsch](CONTRIBUTING-de.md), [English](CONTRIBUTING.md) , [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
```

Comment:
There are some extra space after English and it comma

Reply:
Sorry about that, it slipped my attention... RTL can be a little awkward to edit 😅

Reply:
I agree. And mixing glyph types is a bit weird due to, at least for me, don't know where put the cursor 😥